src/config/configureStore.js

```diff
@@ -26,6 +27,12 @@ function configureStore(history: History) {
     middleware = [...middleware, logger];
   }
 
+  const { REACT_APP_API_URL } = process.env;
+
+  if (REACT_APP_API_URL) {
+    middleware = [...middleware, socketioMiddleware(REACT_APP_API_URL)];
```

Comment:
Why do you rely on the env variable here?
Why not just call `socketioMiddleware()` and use the env variable in the function itself.